### PR TITLE
ref(crons): Clarify what OK and ERROR MonitorStatus are

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Some more documentation is always good

### CODE REVIEW

File: src/sentry/monitors/models.py [hunk 1]
```diff
@@ -92,7 +92,16 @@ class MonitorStatus:
     DELETION_IN_PROGRESS = 3
 
     OK = 4
+    """
+    The monitor environment has recievd check-ins and is OK.
```
Comment (davidenwang) [inline src/sentry/monitors/models.py:96]: nit: received

File: src/sentry/monitors/models.py [hunk 2]
```diff
@@ -92,7 +92,16 @@ class MonitorStatus:
...
+    """
+
     ERROR = 5
+    """
+    The monitor environment is currently in an active incident state. This is a
+    denormalization of thee fact than a MonitorIncident exists for the
+    environment without a resolving cheeck-in.
```
Comment (davidenwang) [inline src/sentry/monitors/models.py:103]: nit: check*

**Review by davidenwang — APPROVED** [review]

File: src/sentry/monitors/models.py (hunk 2 above)
Comment (evanpurkhiser) [reply src/sentry/monitors/models.py:103]: lol typos left and right

File: src/sentry/monitors/models.py [hunk 3]
```diff
@@ -92,7 +92,16 @@ class MonitorStatus:
...
 
     OK = 4
+    """
+    The monitor environment has received check-ins and is OK.
+    """
+
     ERROR = 5
+    """
+    The monitor environment is currently in an active incident state. This is a
+    denormalization of thee fact that a MonitorIncident exists for the
```
Comment (wedamija) [inline src/sentry/monitors/models.py:102]: *thee

**Review by wedamija — APPROVED** [review]

**Pull request merged**